sopel/bot.py

```diff
@@ -578,8 +647,19 @@ def cap_req(self, module_name, capability, arg=None, failure_callback=None,
                 success_callback=None):
         """Tell Sopel to request a capability when it starts.
 
+        :param str module_name: the module requesting the capability
+        :param str capability: the capability requested, optionally prefixed
+                               with ``+`` or ``=``
+        :param str arg: arguments for the capability request
+        :param failure_callback: a function that will be called if the
+                                 capability request fails
+        :type failure_callback: :term:`function`
+        :param success_callback: a function that will be called if the
+                                 capability is successfully requested
+        :type success_callback: :term:`function`
+
         By prefixing the capability with `-`, it will be ensured that the
```

Comment:
Some of them are out of line-note range (GitHub is really pissing me off tonight with that limitation), but every instance of <tt>\`thing\`</tt> in this docstring needs to be replaced with <tt>\`\`thing\`\`</tt>. It's the only place where single-backticks are used (based on searching the generated HTML for incorrect occurrences of `<cite>`) other than the `join` method docstring.

I would also appreciate a general cleanup of any remaining double-spaces after periods, as occurs on what is currently line 671 of the HEAD revision (but that no doubt will have changed by the time you address this note :sweat_smile:)!